android/beagle/src/main/java/br/com/zup/beagle/android/components/utils/RoundedImageView.kt

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-package br.com.zup.beagle.android.view
+package br.com.zup.beagle.android.components.utils
 
+import android.annotation.SuppressLint
 import android.content.Context
 import android.graphics.Canvas
 import androidx.appcompat.widget.AppCompatImageView
-import br.com.zup.beagle.android.utils.applyRadius
 
-internal class BeagleImageView(context: Context) : AppCompatImageView(context) {
-
-    var cornerRadius = 0.0f
+@SuppressLint("ViewConstructor")
+internal class RoundedImageView(context: Context,
```

Comment:
What do you think to use the Kotlin format convention?

```suggestion
internal class RoundedImageView(
    context: Context,
    private val cornerRadius: Double = 0.0
) : AppCompatImageView(context) {
```